lib/rubocop/cop/rspec/factory_bot/consistent_parentheses_style.rb

```diff
@@ -40,6 +40,28 @@ module FactoryBot
         #     name: 'foo'
         #   )
         #
+        # @example `AssignmentOnly: false` (default)
```

Comment:
I'm confused by the `ExplicitOnly` vs `AssignmentOnly` difference. Is this correct?

Reply:
Oops, that is wrong. I corrected it. Thank you.